src/components/cc-feature-list/cc-feature-list.js

```diff
@@ -0,0 +1,212 @@
+    if (this.state.type === 'loading') {
+      return;
+    }
+
+    this.dispatchEvent(new CcFeatureSettingChangeEvent({ featureId: feature.id, newValue }));
+  }
+
+  /**
+   * @param {FeatureStatus} status
+   * @return {{text: string, intent: string}}
+   * @private
+   */
+  _getFeatureStatus(status) {
+    switch (status) {
+      case 'alpha':
+        return { text: i18n('cc-feature-list.status.alpha'), intent: 'warning' };
+      case 'beta':
+        return { text: i18n('cc-feature-list.status.beta'), intent: 'info' };
+      case 'preview':
+        return { text: i18n('cc-feature-list.status.preview'), intent: '' };
+    }
+    return undefined;
+  }
+
+  /**
+   * @param {Feature} feature
+   * @return {TemplateResult}
+   * @private
+   */
+  _renderFeature(feature) {
+    let featureStatus = this._getFeatureStatus(feature.status);
+    return html`
+      <div class="feature">
+        <div class="header">
+          <div class="header-title">
+            <div class="title">${feature.name}</div>
+            <div class="status">
+              ${feature.status
+                ? html` <cc-badge intent="${featureStatus.intent}"> ${featureStatus.text} </cc-badge> `
+                : ''}
+            </div>
+          </div>
+          <cc-toggle
+            .choices="${feature.options}"
+            value="${feature.value}"
+            @cc-select=${(/** @type {CcSelectEvent} */ event) => this._onToggleChange(feature, event.detail)}
+          ></cc-toggle>
+        </div>
+        <div>${feature.description}</div>
+        ${feature.documentationLink != null || feature.feedbackLink != null
+          ? html`
+              <div class="footer">
+                ${feature.documentationLink != null
+                  ? html`
+                      <cc-link
+                        .href=${feature.documentationLink}
+                        a11y-desc="${i18n('cc-feature-list.documentation-link')} - ${feature.name}"
+                        .icon=${iconInfo}
+                      >
+                        ${i18n('cc-feature-list.documentation-link')}
+                      </cc-link>
+                    `
+                  : ''}
+                ${feature.feedbackLink != null
+                  ? html`
+                      <cc-link
+                        .href=${feature.feedbackLink}
+                        a11y-desc="${i18n('cc-feature-list.feedback-link')} - ${feature.name}"
+                        .icon=${iconMessage}
+                      >
+                        ${i18n('cc-feature-list.feedback-link')}
+                      </cc-link>
+                    `
+                  : ''}
+              </div>
+            `
+          : ''}
+      </div>
+    `;
+  }
+
+  render() {
+    return html`
+      <cc-block>
+        <div slot="header-title">${i18n('cc-feature-list.title')}</div>
+        <div slot="content">
+          ${this.state.type === 'loaded' && this.state.featureList.length > 0
```

Comment:
suggestion: with the suggestion below we could change this check to
```suggestion
          ${this.state.type === 'loaded' && !empty
```